docs/contracts.rst

```diff
@@ -99,16 +100,17 @@ This means that cyclic creation dependencies are impossible.
             return new OwnedToken(name);
         }
 
-        function changeName(OwnedToken tokenAddress, bytes32 name) {
+        function changeName(OwnedToken tokenAddress, bytes32 name)  public {
             // Again, the external type of "tokenAddress" is
             // simply "address".
             tokenAddress.changeName(name);
         }
 
-        function isTokenTransferOK(
-            address currentOwner,
```

Comment:
Can you keep the `currentOwner` variable and just comment out its name?

Reply:
I put `currentOwner` back and updated the function `isTokenTransferOK()` to use it and avoid the unused variable warning.